packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -99,6 +99,16 @@ class PouchLink extends CozyLink {
     this.replicationStatus = this.replicationStatus || {}
   }
 
+  /**
+   * Return the PouchDB adapter name.
+   * Should be IndexedDB for newest adapters.
+   *
+   * @returns {string} The adapter name
+   */
+  static getPouchAdapterName = () => {
```

Comment:
nitpick but since it's already a fat arrow function why not use the short syntax `static getPouchAdapterName = () => getAdapterName()`